generators/test/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Generator = require('yeoman-generator');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  writing() {
+    this.fs.copy(
+      this.templatePath('template_test.go'),
+      this.destinationPath('./test/template_test.go')
+    );
+
+    this.fs.copy(
+        this.templatePath('fixture/main.tf'),
+        this.destinationPath('./test/fixture/main.tf')
+      );
+
+    this.fs.copy(
+      this.templatePath('fixture/variables.tf'),
+      this.destinationPath('./test/fixture/variables.tf')
+    );
+
+    this.fs.copy(
+      this.templatePath('fixture/outputs.tf'),
+      this.destinationPath('./test/fixture/outputs.tf')
+    );
```

Comment:
Can we use wildcard matching here? For example, *.tf, *_test.go, *.tfvars?

Reply:
Will try if it supports.

Reply:
Since we need to define file name of destination, I think we should pay attention to all template files and thus it seems not a good idea to use wildcard matching.